common/python_health_helper.sh

```diff
@@ -17,6 +17,11 @@ run() {
   return $res
   }
 
+lint_file() {
+  run "$1 $2" "lint failed on $2"
```

Comment:
Rather than have $1, $2, etc everywhere, please assign $1, ... to named variables with meaningful names.

Reply:
Done